aten/src/ATen/test/pow_test.cpp

```diff
@@ -225,35 +238,79 @@ void tensor_pow_tensor(const Vals vals, c10::ScalarType vals_dtype, Pows pows, c
   }
 }
 
+template<typename T>
+void test_pow_one(const std::vector<T> vals) {
+  for (const auto val : vals) {
+    ASSERT_EQ(native::powi(val, T(1)), val);
+  }
+}
+
+template<typename T>
+void test_squared(const std::vector<T> vals) {
+  for (const auto val : vals) {
+    ASSERT_EQ(native::powi(val, T(2)), val * val);
+  }
+}
+
+template<typename T>
+void test_cubed(const std::vector<T> vals) {
+  for (const auto val : vals) {
+    ASSERT_EQ(native::powi(val, T(3)), val * val * val);
+  }
+}
+template<typename T>
+void test_inverse(const std::vector<T> vals) {
+  for (const auto val : vals) {
+    // 1 has special checks below
+    if ( val != 1 && val != -1) {
+      ASSERT_EQ(native::powi(val, T(-4)), 0);
```

Comment:
Seems like this line should be an exception if we following NumPy.

Reply:
I think there's a comment elsewhere, but we don't follow numpy here (and this PR preserves the behavior on master for integral powers)